### PR TITLE
Homework week2

### PR DESCRIPTION
**Pull request by Farzanna**
sorry for very late uploading, it was really challenging for me

### CODE REVIEW

File: Week2/homework/app.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  console.log(`choose a question by its number
+  1.What is the capital of country X ? (Accept X from user)
+  2.List all the languages spoken in the region Y (Accept Y from user)
+  3.Find the number of cities in which language Z is spoken (Accept Z from user)
+  4.Accept the region and language from the user. Are there any countries in this region with the given language as the official language ? If yes, display those countries. If no, display FALSE. E.g. (A) input region : 'Western Europe' and input language : 'Dutch' output should be Belgium and Netherlands (B) input region : 'Western Europe' and input language : 'Hindi' output should be 'FALSE'
+  5.List all the continents with the number of languages spoken in each continent
+  `);
+}
+
+queriesToBeExecuted = [
```
Comment (mogaleaf) [inline Week2/homework/app.js]: Your queries are all good :)

File: Week2/homework/app.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  'SELECT city.Name FROM city JOIN country ON city.ID = country.Capital WHERE country.Name = ?',
+  'SELECT DISTINCT Language FROM  countrylanguage JOIN country ON CountryCode = Code WHERE Region = ?',
+  'SELECT COUNT(ID) AS "CitiesCount" FROM  countrylanguage JOIN city ON countrylanguage.CountryCode = city.CountryCode WHERE Language = ?',
+  'SELECT country.Name FROM  countrylanguage JOIN country ON CountryCode = Code WHERE Language = ? AND Region = ? AND IsOfficial = "T"',
+  'SELECT Continent, COUNT(DISTINCT Language) AS "numberOfLanguages" FROM  countrylanguage JOIN country ON CountryCode = Code GROUP BY Continent',
+];
+
+async function serviceUserRequest() {
```
Comment (mogaleaf) [inline Week2/homework/app.js]: Super neat

File: Week2/homework/trigger.sql [hunk 3]
```diff
@@ -0,0 +1,18 @@
+
+delimiter $$
+CREATE TRIGGER alertOnCountryCount
```
Comment (mogaleaf) [inline Week2/homework/trigger.sql]: Yes it's good only in this case I would have done the trigger before the insert and not after.

File: Week2/homework/app.js [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+      break;
+    case '5':
+      const continents = await execQuery(queriesToBeExecuted[4]);
+      console.log(continents);
+      break;
+
+    default:
+      console.log('Please insert right request number between 1 and 5.');
+      break;
+  }
```
Comment (mogaleaf) [inline Week2/homework/app.js]: Don't forget to close the connection here.

**Comment by Farzanna** [comment]
> Hello,
> 
> It was challenging but it's a really good homework here ! It's neat and your queries are all good.
> The trigger is working and your ER diagram is Good !
> don't forget to close connection.
> I validate your homework!

Hello @mogaleaf  
thank you for your time and review, I changed it as you mentioned

**Pull request closed without merging**